src/OrchardCore.Modules/OrchardCore.Sms.Azure/AzureSmsPermissionProvider.cs

```diff
@@ -0,0 +1,24 @@
+using OrchardCore.Security.Permissions;
+
+namespace OrchardCore.Sms.Azure;
+
+public class AzureSmsPermissionProvider : IPermissionProvider
+{
+    public static readonly Permission ManageSmsSettings = AzureSmsPermissions.ManageAzureSmsSettings;
```

Comment:
Please remove this from here.

Reply:
Why?

Reply:
Why would you one a static instance to point to another static instance? We had to do this in the past because we needed to move the permission from the permission provider class and into a separate class. In your case, you permission is already in another class, so this isn't needed

Reply:
Maybe because this is an old PR :)

Reply:
yeah. So just remove it then. We don't do that any more in the framework